go.mod

```diff
@@ -1,13 +1,13 @@
 module github.com/cri-o/ocicni
 
-go 1.17
+go 1.20
```

Comment:
**To the reviewer:**

This is needed to be able to update some of our packages that have moved onto the new Go internal API of the Context package:

- https://github.com/golang/go/issues/51365